javascript/javascript1/week1/ez-namey.js

```diff
@@ -0,0 +1,8 @@
+//Ez Namey (Startup name generator)
+ const firstWords = [ "Amazing","super ", "Awesome", "Smart", " Only ","Exciting" , "Nice",
+  "Perfect", " Fascinating"," Digital"];
+const secondWords = ["world", "place", "human",  "Solutions",  "company",  "door",  "value",  " Project ",  "pages",
+  " vision "];
+let companyName =
+  firstWords[Math.floor(Math.random() * 10)] +" " + secondWords[Math.floor(Math.random() * 10)];
```

Comment:
Nice work :) As an improvement, you could use the arrays' lengths instead of "10" so that it will still work if words are added or removed from the arrays:
```suggestion
let companyName =
  firstWords[Math.floor(Math.random() * firstWords.length)] +" " + secondWords[Math.floor(Math.random() * secondWords.length)];
```